src/main/java/org/moera/node/text/markdown/mention/MentionsExtension.java

```diff
@@ -13,7 +13,7 @@
 
     public static final DataKey<NamingCache> NAMING_CACHE = new DataKey<>("NAMING_CACHE", (NamingCache) null);
 
-    private MentionsExtension() {
+    public MentionsExtension() {
```

Comment:
Do you use the constructor anywhere? I see only calls to `create()`. Anyway, if you need it for tests only, package access would be enough.